02.create-and-explore/03.start-working-in-your-sandbox/02.server-settings-for-your-domain/docs.md

```diff
@@ -225,25 +225,44 @@ The reverb settings are:
+## Settings Backup / Restore
+
+You can also download your domain settings on their own. To do so, click **Download Domain Settings**. This will download a .json file onto your computer. ![](domain-settings-backups-1.PNG)
+
+You can upload a settings configuration file to quickly configure a domain. Click on **Choose File**, choose the configuration file from your computer, then click **Upload Domain Settings**. An Are you sure pop-up window will appear. Click on **Restore settings**. Your domain will then restart.
+![](domain-settings-backups-2.PNG) ![](are-you-sure-domain.PNG)
 
 ## Advanced Settings
 
-Additional settings for the Asset Server (ATP), Audio Buffers and Avatar Mixer are available by selecting **Show Advanced** from the sidebar menu. Advanced setting parameters are subject to change. Documentation to come once stabilized.
+Additional settings for the Asset Server (ATP), Audio Buffers and Avatar Mixer are available by selecting the **Advanced Settings** dropdown from each section. Advanced setting parameters are subject to change. Documentation to come once stabilized.
+
+## Content Archives
+
+
+Your domain server makes regular archives of the content in your domain. Content archives includes Models.json, content settings, and assets in [ATP](https://docs.highfidelity.com/create-and-explore/start-working-in-your-sandbox/assignment-clients). You can upload and download content archives onto your computer. From **Content Archives** in the **Content** dropdown menu, you can see and download all of your backups of domain content and content settings. These backups can be used to restore your domain to a previous version or to share your archive and settings info with another user. ![](content-archive-list.png)
+
+To manage automatic content archive intervals, go to **Automatic Content Archives** in the **Settings** dropdown menu or follow the link provided. For more information, see [Automatic Content Archives](https:://docs.highfidelity.com/create-and-explore/start-working-in-your-sandbox/server-settings-for-your-domain#access-your-server-settings). 
+
+You can create a content archive manually by clicking **Generate New Archive**. You should see a Generate a content archive pop-up window. Type in a name for your new archive, then click **Generate Archive**. ![](generate-content-archive.png)
+
+To share your domain content and content settings, simply download a copy of the archive you wish to share and send it to the other user. You can download any archive by clicking the three buttons on the right. ![](download-archive.png)
+
+Finally, you can import a content archive you have stored on your computer by clicking on **Choose File** under Upload Content, selecting the archive .zip file, then clicking **Upload Content**. ![](upload-content.png)
+An Are you sure? pop-up window should appear confirming that your domain content will be replaced. Click **Restore Content**. Your domain server will restart. ![](are-you-sure.png)
```

Comment:
quotes around "Are you sure?"  <-- also be consistent with including ? in these quotes describing pop up window text.  Either use or don't